src/app/visualizations/operators/shareReplay.ts

```diff
@@ -12,33 +12,37 @@ import { take, shareReplay, mergeMapTo } from 'rxjs/operators';
       subscriptions. One interesting note is that this replays from subscription
       to the *subject* Observable rather than the source Observable. If we
       didn't have a visualization for the subject (which creates a
-      subscription), <code>output</code> would emit the same as
+      subscription), <code>delayedOutput</code> would emit the same as
       <code>input</code> other than the initial delay.
     </p>
     <p>
-      The replayed values are emitted all at once. The only difference from
+      Notice how the logs only output twice. These are from the subscriptions
+      that happen for <code>input</code> and <code>subject</code>.
+      <code>delayedOutput</code>, since it's subscribing to our subject, does
+      not output a log.
+    </p>
+    <p>
+      The replayed values are emitted all at once. It's hard to see because the
+      marbles overlap. The only difference from
       <a routerLink="/share"><code>share</code></a> is that
       <code>shareReplay</code> can replay values. <code>share</code> does not
-      replay any values. <code>shareReplay()</code> will replay
+      replay any values. <code>shareReplay</code> will replay
```

Comment:
The parentheses were intentional here to indicate a call to `shareReplay` with no arguments.